packages/eslint-plugin/src/rules/no-input-rename.ts

```diff
@@ -64,13 +68,31 @@ export default createESLintRule<Options, MessageIds>({
       category: 'Best Practices',
       recommended: 'error',
     },
-    schema: [],
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          allowedRenames: {
+            type: 'array',
+            items: {
+              type: 'string',
+            },
+            description: 'A whitelist of allowed input renames',
```

Comment:
We generally don't fill this schema option, but if we do I would rephrase it to:

```suggestion
            description: 'A list with allowed input names',
```

Reply:
Thank you for your review, I found that the description of our rules is directly quoted from codelyzer's page, but there are many rules we have enhanced, so that why I just added this part in the schema. 
I think I can try to add an extra readme document in the `docs` directory for this rule since it already supports additional configuration options. I guess this would make it more intuitive for the user. 😀